content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -404,6 +406,71 @@ instances:
 ### Serverless environment
 
 For more information about AWS Lambda functions, see [how to connect your Lambda telemetry using tags][15].
+
+### OpenTelemetry
+
+When using OpenTelemetry, map the following [resource attributes][16] to their corresponding Datadog conventions:
+
+| OpenTelemetry convention | Datadog convention |
+| --- | --- |
+| `deployment.environment` | `env` |
+| `service.name` | `service` |
+| `service.version` | `version` |
+
+<div class="alert alert-warning">Do not use Datadog-specific tags like <code>DD_SERVICE</code>, <code>DD_ENV</code> or <code>DD_VERSION</code> in your OpenTelemetry configuration.</div>
```

Comment:
I think we should use the term "environment variables" here, and we can rephrase it closer to this:
```suggestion
<div class="alert alert-warning">Datadog-specific environment variables like <code>DD_SERVICE</code>, <code>DD_ENV</code> or <code>DD_VERSION</code> are not supported out of the box in your OpenTelemetry configuration.</div>
```